WIP-CHANGELOG.md

```diff
@@ -65,6 +65,7 @@
 - increased the influxDB pvc size [#739](https://github.com/elastisys/compliantkubernetes-apps/pull/739)
 - Exposed velero's backup timetolive for both sc and wc.
 - disabled internal database for InfluxDB
+- Alertmanager group-by parameters was removed and replaced by an empty list
```

Comment:
Since it's technically not an empty list, I would rather have something like
```suggestion
- Alertmanager group-by parameters was removed and replaced by the special value `...`
    See https://github.com/prometheus/alertmanager/blob/ec83f71/docs/configuration.md#route for more information
```